client/client-lib/src/wallet/mod.rs

```diff
@@ -318,6 +318,10 @@ mod tests {
         ) -> crate::api::Result<Vec<ECashUserBackupSnapshot>> {
             unimplemented!()
         }
+
+        async fn download_client_config(&self) -> crate::api::Result<ConfigResponse> {
+            todo!()
```

Comment:
nit: Should be `unimplement!()`  . AFAIR, `todo` has a meaning - "I was in rush and I'm planning implement it later", while `unimplement` is more "not supported and might never be".